src/Microsoft.AspNet.Proxy/ProxyOptions.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Net.Http;
+
+namespace Microsoft.AspNet.Proxy
+{
+    /// <summary>
+    /// Options to configure host, scheme, and port settings
+    /// </summary>
+    public class ProxyOptions
+    {
+        public string Scheme { get; set; }
+        public string Host { get; set; }
+        public string Port { get; set; }
+        public HttpMessageHandler BackChannelMessageHandler { get; set; }
```

Comment:
I'd be OK changing it in all the Security Middleware too :P. I get what it's doing but I think Outgoing is a clearer name.